pt_BR/readme.md

```diff
@@ -0,0 +1,750 @@
+
+## Dicas
+
+### Arquivos temporários
+
+Executar testes simultaneamente vem com alguns desafios, fazer IO é um deles. Normalmente, os testes de série apenas criam diretórios temporários no diretório de teste atual e o limpam no final. Isso não vai funcionar quando você executar testes simultaneamente, já que esses testes entrarão em conflito uns com os outros. A maneira correta de fazer isso é usar um novo diretório temporário para cada teste. Os módulos [`tempfile`](https://github.com/sindresorhus/tempfile) e [`temp-write`](https://github.com/sindresorhus/temp-write) podem ser úteis.
+
+### Depuração
+
+AVA executa testes simultaneamente por padrão, que é abaixo do ideal quando você precisa depurar algo. Em vez disso, execute os testes em série com a opção `--serial`:
+
+```
+$ ava --serial
+```
+
+### Cobertura de código
+
+Você não pode usar [`istanbul`](https://github.com/gotwarlost/istanbul) para cobertura de código porque o AVA [gera os arquivos de teste](#isolated-environment), mas você pode usar [`nyc`](https://github.com/bcoe/nyc) em vez disso, que é basicamente `istanbul` com suporte para subprocessos.
+
+A partir da versão `5.0.0` ele usa source maps para relatar cobertura para seu código real, independentemente da transpilation. Certifique-se de que o código que você está testando inclui source map inlineou faz referência a um arquivo source map. Se você usar `babel/register`, você pode definir a opção `sourceMaps` em seu `.babelrc` como `inline`.
+
+## Perguntas Frequentes
+
+### Por que não `mocha`, `tape`, `node-tap`?
+
+Mocha requer a utilização de globais implícitas como `describe` e `it` com a interface padrão (que a maioria das pessoas usam), é muito não-opinativo, inchado, síncrono por padrão, tem execução de teste serial, e lento. Tape e node-tap são muito bons. AVA é altamente inspirado em sua sintaxe. No entanto, ambos executam testes em série e fizeram do [TAP](https://testanything.org) um cidadão de primeira classe que, na minha opinião, tornou as suas bases de código um pouco complicadas e acopladas. A saída do TAP é difícil de ler, então você sempre acaba usando um reporter externo. AVA é altamente opinativo e simultâneo. Vem com um simples reporter padrão e suporta TAP através de um sinalizador CLI.
+
+### Como posso utilizar reporters personalizados?
+
+Use a [`--tap` flag](#optional-tap-output) com qualquer [TAP reporter](https://github.com/sindresorhus/awesome-tap#reporters).
+
+### Como o nome é escrito e pronunciado?
+
+AVA, não Ava nem ava. Pronuncia-se [`/ˈeɪvə/` ay-və](media/pronunciation.m4a?raw=true).
+
+### O que é o plano de fundo do cabeçalho?
+
+[Galáxia de Andrômeda.](https://simple.wikipedia.org/wiki/Andromeda_galaxy)
```

Comment:
This link is better ? (https://pt.wikipedia.org/wiki/Gal%C3%A1xia_de_Andr%C3%B4meda)